client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -52,35 +57,81 @@ const EditPage = createClass({
 
 	getInitialState : function() {
 		return {
-			brew                   : this.props.brew,
+			brew : {
+				text        : this.props.brew.text || '',
+				style       : this.props.brew.style || '',
+				shareId     : this.props.brew.shareId || null,
+				editId      : this.props.brew.editId || null,
+				gDrive      : this.props.brew.gDrive || false,
+				title       : this.props.brew.title || '',
+				description : this.props.brew.description || '',
+				tags        : this.props.brew.tags || '',
+				published   : this.props.brew.published || false,
+				authors     : this.props.brew.authors || [],
+				systems     : this.props.brew.systems || [],
+				renderer    : this.props.brew.renderer || 'legacy'
+			},
 			isSaving               : false,
 			isPending              : false,
 			alertTrashedGoogleBrew : this.props.brew.trashed,
 			alertLoginToTransfer   : false,
-			saveGoogle             : this.props.brew.googleId ? true : false,
+			saveGoogle             : (global.account && global.account.googleId ? true : false),
```

Comment:
```suggestion
			saveGoogle             : this.props.brew.googleId || (global.account?.googleId && this.props.editType == 'new') ? true : false,
```
We want to load the page with `saveGoogle` true in two cases:
1) The file is already saved to Google (this.props.brew.googleId exists)
2) It is a new page (editType = 'new') and the user is signed in with Google

This currently only sets `saveGoogle` true if the user is signed in with Google. It will ignore whether the brew was previously saved to Google, and if a user visits the edit page accidentally without being logged in, any save will transfer the brew to HB.